### ISSUE TITLE
Support clickers

### ISSUE THREAD

**Issue by kettanaito** [open-draft/presenteer#7]
Presenteer should be operational by clickers (remote devices to navigate through the presentation).

## Roadmap

1. Check which event a clicker produces.
1. Ensure the produced event(s) are listened to, or piped to the existing navigation

**Comment by kettanaito** [comment]
Here's a great [reference of slides navigation](https://github.com/jxnblk/mdx-deck/blob/master/packages/gatsby-theme/src/hooks/use-keyboard.js) from `mdx-deck`.
